examples/eks-cluster-with-observability/README.md

```diff
@@ -0,0 +1,146 @@
+# EKS Cluster with Observability Tools
+
+This example deploy a new Kubernetes Cluster with Amazon Managed Prometheus, Amazon Managed Grafana, FluentBit, and Opensearch. It also includes instructions for deploying a sample workload with ArgoCD to generate logs and metrics.
+
+---
+**NOTE**
+
+For the sake of simplicity in this example, we store sensitive information and credentials in `dev.tfvars`. This should not be done in a production environment. Instead, store the information in AWS Secrets Manager.
+
+---
+
+## How to Deploy
+
+### Prerequisites
+
+- An existing Amazon Managed Grafana (AMG) Workspace.
+  - As of this writing (January 25, 2022), the AWS Terraform Provider does not support Amazon Managed Grafana, so it must be manually created beforehand. Instructions [here](https://docs.aws.amazon.com/grafana/latest/userguide/getting-started-with-AMG.html)
+
+#### Generate a Grafana API Key
+- Give the SSO user you set up in when creating the AMG Workspace admin access. 
+  - In the AWS Console, navigate to Amazon Grafana. In the left navigation bar, click __All workspaces__, then click __eks-ssp-observability__
```

Comment:
where does `_eks-ssp-observability__` come from?

Reply:
Whoops - this is the workspace name I used when I created it in AMG. I'll update this to "then click on the workspace name you are using for this example."